internal/batch/policy/policy_test.go

```diff
@@ -127,6 +128,50 @@ func TestPolicyPeriod(t *testing.T) {
 	}
 }
 
+func TestPolicyPeriodJitter(t *testing.T) {
+	conf := batchconfig.NewConfig()
+	conf.Period = "100ms"
+	conf.Jitter = 0.5
+
+	pol, err := policy.New(conf, mock.NewManager())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	tCtx, done := context.WithTimeout(context.Background(), time.Second*30)
+	t.Cleanup(func() {
+		require.NoError(t, pol.Close(tCtx))
+		done()
+	})
+
+	// Set RNG seed to ensure deterministic test runs
+	//nolint:staticcheck
+	rand.Seed(1)
```

Comment:
```suggestion
	rand.New(rand.NewSource(1))
```

don't think it's a big deal